common/djangoapps/django_comment_common/migrations/0003_auto_20161107_0954.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+from django.db import migrations, models
+
+def add_default_enable(apps, schema_editor):
+    ForumsConfig = apps.get_model("django_comment_common", "ForumsConfig")
+    settings_count = ForumsConfig.objects.count()
+    if settings_count is 0:
+        settings = ForumsConfig(enabled=True)
```

Comment:
Okay, maybe I'm misunderstanding what this is doing. My goal with this statement is that if you run this migration and forums were enabled, they don't suddenly disappear from the user's perspective. If they were disabled (because it's a sandbox, or your install doesn't use forums), then running this migration shouldn't suddenly enable them.
If the present code already fulfills both of these, then please add a comment explaining that for folks like me who might have the same misunderstanding.